src/strategies/eventbrite.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  Ctor: require('passport-eventbrite-oauth').OAuth2Strategy,
+  getConfig: (env, callbackURL) => {
+    const clientID = env.LW_EVENTBRITE_CLIENTID
+    const clientSecret = env.LW_EVENTBRITE_CLIENTSECRET
+    if (clientID && clientSecret) {
+      return {
+        clientID,
+        clientSecret,
+        callbackURL
+      }
+    }
+  },
+
+  toUser: (accessToken, refreshToken, profile, done) => {
+    const {username, displayName, emails} = profile
+    done(null, {
+      accessToken,
+      refreshToken,
+      profile: {
+        username,
+        name: displayName,
+        provider: 'eventbrite',
+        email: emails[0].value
```

Comment:
up to now, email is not included in public profile, since it it considered private information. the profile is not http-only and thus is more likely to be subject to an evil script / "attack".